packages/atomic/src/components/search/atomic-breadbox/atomic-breadbox.tsx

```diff
@@ -84,12 +85,18 @@ export class AtomicBreadbox implements InitializableComponent {
   private breadcrumbShowLessFocus!: FocusTargetController;
 
   public initialize() {
-    this.breadcrumbManager = buildBreadcrumbManager(this.bindings.engine);
-    this.facetManager = buildFacetManager(this.bindings.engine);
-
-    if (window.ResizeObserver) {
-      this.resizeObserver = new ResizeObserver(() => this.adaptBreadcrumbs());
-      this.resizeObserver.observe(this.host.parentElement!);
+    if (!closest(this.host, 'atomic-search-interface')) {
```

Comment:
Maybe the bindings could contain what type of interface you're connected to (search/insight)